cxx-qt-gen/src/generator/naming/property.rs

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+use crate::generator::naming::CombinedIdent;
+use crate::parser::property::ParsedQProperty;
+use convert_case::{Case, Casing};
+use quote::format_ident;
+use syn::Ident;
+
+/// Names for parts of a Q_PROPERTY
+pub struct QPropertyName {
+    pub name: CombinedIdent,
+    pub getter: CombinedIdent,
+    pub setter: CombinedIdent,
+    pub notify: CombinedIdent,
+}
+
+impl From<&ParsedQProperty> for QPropertyName {
+    fn from(property: &ParsedQProperty) -> Self {
+        let ident = &property.ident;
+        Self {
+            name: name_from_ident(ident),
+            getter: getter_from_ident(ident),
+            setter: setter_from_ident(ident),
+            notify: notify_from_ident(ident),
+        }
+    }
+}
+
+/// For a given ident generate the Rust and C++ getter names
+pub fn getter_from_ident(ident: &Ident) -> CombinedIdent {
```

Comment:
These function shouldn't be `pub` imho.
Just create a QPropertyName once, then access the different members.